jina/flow/base.py

```diff
@@ -364,7 +363,7 @@ def __init__(
         self.k8s_infrastructure_manager = None
         if self.args.infrastructure == InfrastructureType.K8S:
             self.k8s_infrastructure_manager = self._FlowK8sInfraResourcesManager(
-                k8s_namespace=self.args.name,
+                k8s_namespace=self.args.k8s_namespace or self.args.name,
```

Comment:
I would create a local variable out of `self.args.k8s_namespace or self.args.name` for more compact code

Reply:
@florian-hoenicke change everywhere where I use `or` or only this specific line?